### ISSUE TITLE
Redux's bindActionCreators make the state logs unreadable

### ISSUE THREAD

**Issue by gouegd** [gaearon/redux-devtools#124]
Hi there,

**TLDR**: using `bindActionCreators` confuses the devtools

I was having an issue after setting up redux devtools: it all seemed good but all the logs appearing in the devtools console had a state object with lots of functions, and that was nothing like I expected (a simple counter value - my redux app is the universal redux sample from a few days ago).
I removed the use of `bindActionCreators`, I'm passing all actions to the smart component in the `mapStateToProps` function, and make use of `dispatch` to... dispatch the actions, instead of relying on the bound magic.
The result is it works fine, the state logged in devtools is now correct.
So, my opinion after looking at the source from `bindActionCreators` (seems fine, it's simple enough!), is that the current default setup for the redux devtools does not handle this.

**Comment by gaearon** [comment]
Sorry, but I don't really understand what issue you are talking about. The TodoMVC example included with `redux-devtools` uses `bindActionCreators` and does not appear to suffer the problem you described.

Please provide a reproducible example and I'll reopen the issue.

**Comment by gouegd** [comment]
Right, I think it was due to the fact redux-logger was also in my middleware pipe at the time. Thanks for your attention anyway !
